src/airodump-ng/airodump-ng.c

```diff
@@ -6688,6 +6737,13 @@ int main(int argc, char * argv[])
 		return (EXIT_FAILURE);
 	}
 
+	if (lopt.ignore_other_channels && !lopt.chanoption)
+	{
+		printf("Notice: specify \"--ignore-other-chans\" with \"--channel\"\n");
```

Comment:
Wording should be updated: "Error: --ignore-other-chans requires --channel (or -c)

Reply:
updated, please check again